### PR TITLE
DOC-2327 TinyMCE 7.0 documentation clean up

### PR DESCRIPTION
**Pull request by FarzadHayat**
Ticket: DOC-2327

Site: [DOC-2327 site](http://docs-feature-70-doc-2327.staging.tiny.cloud/docs/tinymce/latest/7.0-release-notes)

Changes:
* See `Files changed` tab

Pre-checks:
- [x] Branch prefixed with `feature/7/` or `hotfix/7/`
- [x] `modules/ROOT/nav.adoc` has been updated `(if applicable)`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 1]
```diff
@@ -484,38 +508,31 @@ However, the `aria-placeholder` attribute is not a valid attribute for the body
...
+As of {productname} 7.0, the tooltip property is now optional for `DialogFooterToggleButton`. This flexibility allows developers to omit the tooltip for buttons where it's not required, providing a cleaner and more focused user experience.
 
-For further details on the `+sandbox_iframes+` option, see the xref:content-filtering.adoc#sandbox-iframes[the content filtering options], or refer to the xref:security.adoc#sandbox-iframes[security guide], or the link:https://www.tiny.cloud/docs/tinymce/6/6.8.1-release-notes/#new-sandbox_iframes-option-that-controls-whether-iframe-elements-will-be-added-a-sandbox-attribute-to-mitigate-malicious-intent[{productname} 6.8.1 release notes].
+=== Changed the `media_url_resolver` option to use promises.
+// #TINY-9154
 
-=== Remove the height field from the `table` plugin cell dialog. The `table` plugin row dialog now controls the row height by setting the height on the `tr` element, not the `td` elements.
-//# TINY-10617
+// TODO
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.0-release-notes.adoc:521]: @ArvinJ-H Need a release note entry for TINY-9154.
You can make a commit directly to this ticket.

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 2]
```diff
@@ -549,81 +566,70 @@ NOTE: The new `trigger` option can handle multiple character strings as the trig
 
 For more information, visit the updated xref:autocompleter.adoc[Autocompleter] documentation.
 
-=== The open-source `template` plugin has been removed.
-// #TINY-10654
+=== Removed `closeButton` from `NotificationSpec`, close button in notification is now rendered by default.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.0-release-notes.adoc:569]: @ShiridiGandham can you please review this newly added release note entry for TINY-10646?
See preview here: http://docs-feature-70-doc-2327.staging.tiny.cloud/docs/tinymce/latest/7.0-release-notes/#removed-closebutton-from-notificationspec-close-button-in-notification-is-now-rendered-by-default

Comment (FarzadHayat) [reply modules/ROOT/pages/7.0-release-notes.adoc:569]: - [ ] TINY-10646 reviewed by @ShiridiGandham

File: modules/ROOT/pages/7.0-release-notes.adoc (hunk 1 above)
Comment (FarzadHayat) [reply modules/ROOT/pages/7.0-release-notes.adoc:521]: - [ ] Release note added for TINY-9154 by @ArvinJ-H

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 3]
```diff
@@ -484,38 +508,31 @@ However, the `aria-placeholder` attribute is not a valid attribute for the body
...
+As of {productname} 7.0, the tooltip property is now optional for `DialogFooterToggleButton`. This flexibility allows developers to omit the tooltip for buttons where it's not required, providing a cleaner and more focused user experience.
 
-To prevent any expected iframes from being sandboxed, we recommend adding the source domains of such iframes to the new xref:content-filtering.adoc#sandbox-iframes-exclusions[`+sandbox_iframes_exclusions+`] option list, and including the domains in the default list where necessary. To prevent all iframes from being sandboxed, set the option `+sandbox_iframes+` to `+false+` in your editor configuration.
+=== Changed the `media_url_resolver` option to use promises.
+// #TINY-9154
 
-For further details on the `+sandbox_iframes+` option, see the xref:content-filtering.adoc#sandbox-iframes[the content filtering options], or refer to the xref:security.adoc#sandbox-iframes[security guide], or the link:https://www.tiny.cloud/docs/tinymce/6/6.8.1-release-notes/#new-sandbox_iframes-option-that-controls-whether-iframe-elements-will-be-added-a-sandbox-attribute-to-mitigate-malicious-intent[{productname} 6.8.1 release notes].
+// TODO
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.0-release-notes.adoc:521]: - [ ] Release note added for TINY-9154

File: modules/ROOT/pages/7.0-release-notes.adoc (hunk 2 above)
Comment (ShiridiGandham) [reply modules/ROOT/pages/7.0-release-notes.adoc:569]: LGTM

File: modules/ROOT/pages/7.0-release-notes.adoc (hunk 3 above)
Comment (FarzadHayat) [reply modules/ROOT/pages/7.0-release-notes.adoc]: - [ ] add demo to support release note entry

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 4]
```diff
@@ -484,38 +508,33 @@ However, the `aria-placeholder` attribute is not a valid attribute for the body
...
+=== Changed the `media_url_resolver` option to use promises.
+// #TINY-9154
 
-For further details on the `+sandbox_iframes+` option, see the xref:content-filtering.adoc#sandbox-iframes[the content filtering options], or refer to the xref:security.adoc#sandbox-iframes[security guide], or the link:https://www.tiny.cloud/docs/tinymce/6/6.8.1-release-notes/#new-sandbox_iframes-option-that-controls-whether-iframe-elements-will-be-added-a-sandbox-attribute-to-mitigate-malicious-intent[{productname} 6.8.1 release notes].
+In {productname} 7.0, the `media_url_resolver` has been refactored to use promises instead of callbacks. This change brings standardization to the code base and improves error handling.
 
-=== Remove the height field from the `table` plugin cell dialog. The `table` plugin row dialog now controls the row height by setting the height on the `tr` element, not the `td` elements.
-//# TINY-10617
+// TODO: demo to show the function changing from `(data, resolve, reject) => void` to `(data) => Promise<result>`
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.0-release-notes.adoc:523]: - [ ] add demo to support TINY-9154

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 5]
```diff
@@ -484,38 +508,33 @@ However, the `aria-placeholder` attribute is not a valid attribute for the body
...
 
-To prevent any expected iframes from being sandboxed, we recommend adding the source domains of such iframes to the new xref:content-filtering.adoc#sandbox-iframes-exclusions[`+sandbox_iframes_exclusions+`] option list, and including the domains in the default list where necessary. To prevent all iframes from being sandboxed, set the option `+sandbox_iframes+` to `+false+` in your editor configuration.
+=== Changed the `media_url_resolver` option to use promises.
+// #TINY-9154
 
-For further details on the `+sandbox_iframes+` option, see the xref:content-filtering.adoc#sandbox-iframes[the content filtering options], or refer to the xref:security.adoc#sandbox-iframes[security guide], or the link:https://www.tiny.cloud/docs/tinymce/6/6.8.1-release-notes/#new-sandbox_iframes-option-that-controls-whether-iframe-elements-will-be-added-a-sandbox-attribute-to-mitigate-malicious-intent[{productname} 6.8.1 release notes].
+In {productname} 7.0, the `media_url_resolver` has been refactored to use promises instead of callbacks. This change brings standardization to the code base and improves error handling.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.0-release-notes.adoc:521]: - [ ] @ArvinJ-H review this please

Comment (ArvinJ-H) [reply modules/ROOT/pages/7.0-release-notes.adoc:521]: LGTM

**Review by ArvinJ-H — APPROVED** [review]

**Review by danoaky-tiny — APPROVED** [review]

**Review by shanmen-tiny — APPROVED** [review]

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 6]
```diff
@@ -748,66 +748,89 @@ Previously in {productname}, there was an issue with `selection.setContent` when
...
-//#TINY-vwxyz1
+[[known-issues]]
+== Known issues
 
-// CCFR here.
+This section describes issues that users of {productname} 7.0 may encounter and possible workarounds for these issues.
 
-[[deprecated]]
-== Deprecated
+There are two known issues in {productname} 7.0.
```
Comment (kemister85) [inline modules/ROOT/pages/7.0-release-notes.adoc:796]:
```suggestion
There are four known issues in {productname} 7.0.
```

**Review by MitchC1999 — APPROVED** [review]

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 7]
```diff
@@ -748,66 +748,89 @@ Previously in {productname}, there was an issue with `selection.setContent` when
...
-=== Revision history
+Currently, tab navigation within the Enhanced Code Editor is not working as expected when Safari is the host browser and the editor configuration is set to `advcode_inline: true`, resulting in a loss of tab functionality.
+
+.Example of configuration affected.
+[source, js]
+----
+tinymce.init({
+  selector: 'textarea', // change this value according to your HTML
+  plugins: 'advcode',
+  advcode_inline: true,
```
Comment (kemister85) [inline modules/ROOT/pages/7.0-release-notes.adoc:826]:
```suggestion
```

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 8]
```diff
@@ -716,102 +730,106 @@ As a consequence, this would result in the menubar and/or the toolbar to be squa
...
+TODO//
 
-This section describes issues that users of {productname} <x.y[.z]> may encounter and possible workarounds for these issues.
+Currently, there is no workaround for this issue.
 
-There <is one | are <number> known issue<s> in {productname} <x.y[.z]>.
+=== Revision History has no integration for saving version.
 
-=== Revision history
+TODO//
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/7.0-release-notes.adoc:828]:
```suggestion
```

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 9]
```diff
@@ -716,102 +730,106 @@ As a consequence, this would result in the menubar and/or the toolbar to be squa
...
 
-[[known-issues]]
-== Known issues
+TODO//
 
-This section describes issues that users of {productname} <x.y[.z]> may encounter and possible workarounds for these issues.
+Currently, there is no workaround for this issue.
 
-There <is one | are <number> known issue<s> in {productname} <x.y[.z]>.
+=== Revision History has no integration for saving version.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/7.0-release-notes.adoc:826]:
```suggestion
=== Revision History has no integration for saving a version.
```

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 10]
```diff
@@ -716,102 +730,106 @@ As a consequence, this would result in the menubar and/or the toolbar to be squa
...
+// #TINY-10596
 
-// CCFR here.
+Currently, tab navigation within the Enhanced Code Editor is not working as expected when Safari is the host browser, resulting in a loss of tab functionality.
 
-[[deprecated]]
-== Deprecated
+Currently, there is no workaround for this issue.
 
+=== Revision History highlights `table` content with `colgroup` as new.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/7.0-release-notes.adoc:820]:
```suggestion
=== table that has `colgroup` attribute is highlighted as new in Revision History.
```

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 11]
```diff
@@ -716,102 +730,106 @@ As a consequence, this would result in the menubar and/or the toolbar to be squa
...
 
-[[known-issues]]
-== Known issues
+TODO//
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/7.0-release-notes.adoc:822]:
```suggestion
When a `table` element has `colgroup` attribute, Revision History highlights it as new even though it has no change.
```

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/ROOT/pages/7.0-release-notes.adoc (hunk 4 above)
Comment (kemister85) [reply modules/ROOT/pages/7.0-release-notes.adoc]: outdated, i linked this to the xref:media.adoc#media_url_resolver[media_url_resolver] page, and added entry to the migration guide.

**Review by kemister85 — APPROVED** [review]

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 12]
```diff
@@ -716,102 +730,105 @@ As a consequence, this would result in the menubar and/or the toolbar to be squa
...
+// #TINY-10596
 
-// CCFR here.
+Currently, tab navigation within the Enhanced Code Editor is not working as expected when Safari is the host browser, resulting in a loss of tab functionality.
 
-[[deprecated]]
-== Deprecated
+Currently, there is no workaround for this issue.
 
+=== table that has `colgroup` attribute is highlighted as new in Revision History.
```
Comment (kemister85) [inline modules/ROOT/pages/7.0-release-notes.adoc:820]:
```suggestion
=== Table that has `colgroup` attribute is highlighted as new in Revision History.
```

File: modules/ROOT/partials/configuration/remove_trailing_brs.adoc [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+
+=== Example: using `+remove_trailing_brs+`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  remove_trailing_brs: false
+});
+----
```
Comment (kemister85) [inline modules/ROOT/partials/configuration/remove_trailing_brs.adoc:22]: @MitchC1999 I have added `remove_trailing_brs` option back to the docs, lets discuss updating this page to suit changes.

**Pull request merged**